pyfmt/__init__.py

```diff
@@ -25,19 +28,139 @@
+        )
+
+
+def pyfmt(
+    path, skip="", isort=False, black=False, check=False, line_length=100, show_title=False, extra_isort_args="", extra_black_args=""
+) -> int:
     """Run isort and black with the given params and print the results."""
+    if show_title:
+        pyfmt_title()  # Display title
+    timer_start = time.time()  # Measure how long everything takes
+
+    if skip:
+        # Map out current working directory
+        all_files, all_dirs = find_all_files_and_dirs()
+        # If specified files and directories exist, store the filenames
+        skips = skip.split(",")
+        filenames_to_skip = []
+        for item in skips:
+            if item in all_files:
+                if item.split(".")[-1] == "py":
+                    filenames_to_skip.extend([item])
+            elif os.path.abspath(item) in all_dirs:
+                # Saving all filenames in user specified directory
+                files_in_dir = []
+                for (dirpath, dirnames, filenames) in os.walk(item):
+                    for filename in filenames:
+                        if filename.split(".")[-1] in ("py", "pyi"):
+                            files_in_dir.append(filename)
+                filenames_to_skip.extend(files_in_dir)
+            else:
+                print(f'CRITICAL: One of the files or directories marked as skipped not found ("{item}").')
+                print("CRITICAL: Check spelling or existence of file or directory")
+                print("CRITICAL: Aborting pyfmt ...")
+                sys.exit()
+        # Display Files skipped
+        display_divider(title="SKIPPING FILES")
+        for filename_to_skip in filenames_to_skip:
+            print(f"SKIPPING: {filename_to_skip}")
+        print(f"\nNumber of files to be skipped by pyfmt: {len(filenames_to_skip)}")
+        # Make a continuos string of arguments for
+        #   isort - must be separate --skip for each file
+        #   black - regex for exact filename (ie. file1|file2|etc.)
+        isort_filenames_to_skip = "--skip=" + " --skip=".join(filenames_to_skip)
+        black_filenames_to_skip = "--exclude=" + "|".join(filenames_to_skip)
+        # Adding to the isort and black arguments
+        extra_isort_args += isort_filenames_to_skip
+        extra_black_args += "--exclude=" + black_filenames_to_skip
+
     if check:
         extra_isort_args += " --check-only"
         extra_black_args += " --check"
 
-    isort_exitcode = run_formatter(
-        ISORT_CMD, path, line_length=line_length, extra_isort_args=extra_isort_args
-    )
-    black_exitcode = run_formatter(
-        BLACK_CMD, path, line_length=line_length, extra_black_args=extra_black_args
-    )
+    # If user specified to run only isort or black
+    isort_black_run = [True, True]
+    if isort:
+        isort_black_run = [True, False]
+    if black:
+        isort_black_run = [False, True]
+
+    # Executing isort import formatter
+    isort_exitcode = 0
+    if isort_black_run[0]:
```

Comment:
why not use `isort`?  (should be `isort_only`)

same for below, why not just use `black` (should be `black_only`)?